internal/graph/graph.go

```diff
@@ -416,60 +416,3 @@ func (g *RelationshipGraph) getRelationshipEdgesWithTargetRewrite(
 		panic("unexpected userset rewrite encountered")
 	}
 }
-
-// NewLayeredCheckResolver constructs a CheckResolver that is composed of various CheckResolver layers.
-// Specifically, it constructs a CheckResolver with the following composition:
-//
-//	CycleDetectionCheckResolver  <-----|
-//		CachedCheckResolver              |
-//			DispatchThrottlingCheckResolver |
-//				LocalChecker                   |
-//					CycleDetectionCheckResolver -|
-//
-// The returned CheckResolverCloser should be used to close all resolvers involved in the
-// composition after you are done with the CheckResolver.
-func NewLayeredCheckResolver(
```

Comment:
For other reviewers: the point of this PR was to get rid of this function, which has duplicate logic as the `NewServer` function: https://github.com/openfga/openfga/blob/518606928ded775a86a6ed69c4e6faa245e9b372/pkg/server/server.go#L496-L546